crates/publish/src/expander.rs

```diff
@@ -98,24 +92,27 @@ fn bindle_manifest(
     })
 }
 
-fn bindle_component_manifest(
+async fn bindle_component_manifest(
     local: &local_schema::RawComponentManifest,
     base_dir: &Path,
-) -> Result<bindle_schema::RawComponentManifest> {
+) -> PublishResult<bindle_schema::RawComponentManifest> {
     let source_digest = match &local.source {
         local_schema::RawModuleSource::FileReference(path) => {
             let full_path = base_dir.join(path);
-            file_sha256_string(&full_path)
-                .with_context(|| format!("Failed to get parcel id for '{}'", full_path.display()))?
+
+            if let Ok(false) = Path::try_exists(&full_path) {
```

Comment:
Do you have a min Rust version compiler requirement?
`try_exists` is marked stable from 1.63.0 (https://doc.rust-lang.org/stable/std/path/struct.Path.html#method.try_exists)

Reply:
1.63 shouldn't be a problem.  I think CI tracks latest stable.